modules/custom/cu_shortcodes/cu_shortcodes.module

```diff
@@ -783,7 +792,7 @@ function cu_shortcodes_map($attrs, $content = NULL) {
  * Map tips.
  */
 function cu_shortcodes_map_tips($format, $long) {
-  $output = '[map]https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d3054.215386660962!2d-105.26572850973943!3d40.04829405429418!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x876beef682730725%3A0xc7604d66a96a0f1e!2sNorth+Boulder!5e0!3m2!1sen!2sus!4v1400010841644[/map]';
+  $output = '[map]YOUR GOOGLE MAPS EMBED URL[/map]';
```

Comment:
This can take maps from map.colorado.edu as well as google maps. There are also sizes (small, medium, large). So it might be [map size="small/medium/large"]YOUR MAP EMBED URL HERE[/map] something like that?